### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by marquiz**
- Update golang to v1.20.4
- go.mod: update deps
  Update kubernetes to v1.25.10. Update github.com/docker/distribution
(indirect dep) to the latest release.

### CODE REVIEW

**Review by klihub — APPROVED** [review]
LGTM.

File: go.mod [hunk 1]
```diff
@@ -48,7 +48,7 @@ require (
 	github.com/coreos/go-systemd/v22 v22.3.2 // indirect
 	github.com/cyphar/filepath-securejoin v0.2.3 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/docker/distribution v2.8.1+incompatible // indirect
+	github.com/docker/distribution v2.8.2+incompatible // indirect
```
Comment (askervin) [inline go.mod:51]: hooray

**Review by askervin — APPROVED** [review]
LGTM

**Review by jukkar — APPROVED** [review]

**Comment by marquiz** [comment]
Who dares to merge this?

**Pull request merged**